### DISCUSSION TITLE
Do Receiver nodes not work with a ManagerSearch node?

### DISCUSSION THREAD

**Discussion by technox123** [Security-Onion-Solutions/securityonion#12472] [2.4] [answered]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

64

### Storage for /

500gb

### Storage for /nsm

1tb

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hi,

Like the title of this discussion. Do receiver nodes not work with a managersearch node? Do you have to have a dedicated search node?

I'm asking this because I just tried installing the receiver node in my distributed install (I have 1x Forward node and 1x Managersearch Node) and it seems like the redis queue is not going anywhere on the receiver node, it's just consistently rising.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by m0duspwnens** [comment]
@technox123 Thanks for the report. I will try to look into this sometime soon.

**Comment by m0duspwnens** [comment, marked as answer]
Thanks again for the report. The issue has been resolved and will be in the next release. https://github.com/Security-Onion-Solutions/securityonion/issues/12535
